### ISSUE TITLE
[HOLD #16078] Chat - Disappears popup mentioning @here when clicking outside the compos box

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#20128]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
**Issue found when executing Pr** https://github.com/Expensify/App/pull/19880
## Action Performed:
1. Open the App
2. Go to any chat
2. Type  `@here`
3. Click outside popup option
4. Click again on the compose box

## Expected Result:
Window with pop-up options appears, after user touches the compose box

## Actual Result:
Disappears popup mentioning  `@here` when clicking outside the compos box

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [x] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:**1.3.23.6

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/eedd49ed-47e2-46a3-aac8-8e8fe43e6011

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mallenexpensify** [comment]
@allroundexperts good job in your explanation under "What is the root cause of that problem?"
I agree the issue is 'where' you click in the compose box. ie. if you click on whatever @[any-email] was highlighted, it'll likely show the popup.  When you click a blank area in the compose box the popup might not show.  

@dangrous 👀 plz.  I'm unsure what's best

**Comment by bernhardoj** [comment]
This has the same root cause with this issue https://github.com/Expensify/App/issues/16364 and probably the behavior gonna change based on this comment https://github.com/Expensify/App/issues/16364#issuecomment-1576895962.

**Comment by dangrous** [comment]
So I don't think this is a feature that most people would expect (most apps don't do this) BUT we do seem to do it on other platforms and our goal is always consistency between the platforms so I think we should either fix it here, or remove it everywhere (which is also what the comment @bernhardoj linked to suggests).

Not sure if that's an answer or not. Something should be done, just not sure what. Maybe we should discuss in Slack, or hold on 16364 and see what gets decided there?

**Comment by dangrous** [comment]
What do you think about ^ @mallenexpensify ? I'm happy to bring it up in #expensify-open-source if you agree, or we can just hold on #16364 and see what happens, and adjust this accordingly.

**Comment by mallenexpensify** [comment]
Thanks @dangrous , that was helpful.  My take is.... this isn't a priority item to fix (regardless if it's a bug or new feature).  Since there are other related issues, I propose we put this on hold pending 
- https://github.com/Expensify/App/issues/16078

and also on 
- https://github.com/Expensify/App/issues/16364

Bumping to weekly and removing `Bug` since it's going on hold

**Comment by dangrous** [comment]
Yep, that works for me!

**Comment by dangrous** [comment]
still on hold

**Comment by dangrous** [comment]
#16078 is still in progress, so we're waiting

**Comment by dangrous** [comment]
Same as above

**Comment by dangrous** [comment]
Looks like we're still progressing on the linked issue, but no action needed here yet.

**Comment by dangrous** [comment]
Still on hold

**Comment by dangrous** [comment]
Same as above

**Comment by dangrous** [comment]
Still holding

**Comment by dangrous** [comment]
holdy holdy. Looks like https://github.com/Expensify/App/issues/16078 was waiting on a React Native upstream update, and that's now merged. So we'll get there.

**Comment by dangrous** [comment]
Progress is being made on blocking issue, but we're still not there yet

**Comment by dangrous** [comment]
Same as above

**Comment by mallenexpensify** [comment]
Still on hold
https://github.com/Expensify/App/issues/16078

**Comment by dangrous** [comment]
Still waitin on https://github.com/Expensify/App/issues/16078

**Comment by mallenexpensify** [comment]
held on upstream fix

**Comment by dangrous** [comment]
same as above

**Comment by dangrous** [comment]
Still waiting

**Comment by dangrous** [comment]
One day we'll get there, still holding

**Comment by dangrous** [comment]
same as above

**Comment by dangrous** [comment]
Continue to hold

**Comment by dangrous** [comment]
I think the held issue is getting there

**Comment by dangrous** [comment]
The issue this is held on seems like we're almost done?

**Comment by dangrous** [comment]
Or not? Still on hold, I think upstream changes take quite some time

**Comment by dangrous** [comment]
Same as above

**Comment by dangrous** [comment]
Still holding

**Comment by dangrous** [comment]
Work continues on #16078

**Comment by dangrous** [comment]
Same as above

**Comment by dangrous** [comment]
Same.

**Comment by dangrous** [comment]
fix for #16078 looks like it's in review, so we're nearly there! Then we have to remember what this issue even is, because I sure don't.

**Comment by dangrous** [comment]
we'll get there soon, hopefully

**Comment by dangrous** [comment]
Really just for my own purposes, [this](https://github.com/Expensify/App/pull/35149) is the PR we're waiting for!

**Comment by mallenexpensify** [comment]
on hold...

**Comment by mallenexpensify** [comment]
Still holding on
- https://github.com/Expensify/App/issues/16078#issuecomment-1963578845

**Comment by mallenexpensify** [comment]
Still holding
- https://github.com/Expensify/App/issues/16078 
-

**Comment by mallenexpensify** [comment]
Still on hold

**Comment by mallenexpensify** [comment]
Held
- https://github.com/Expensify/App/issues/16078

**Comment by mallenexpensify** [comment]
Not much movement on 
- https://github.com/Expensify/App/issues/16078

**Comment by mallenexpensify** [comment]
On hold

**Comment by mallenexpensify** [comment]
Hold still
- https://github.com/Expensify/App/issues/16078

**Comment by dangrous** [comment]
forever on hold

**Comment by dangrous** [comment]
Thanks for [checking in](https://github.com/Expensify/App/issues/16078#issuecomment-2148623301) on that other issue @mallenexpensify!

**Comment by dangrous** [comment]
Looks like the PR that fixes #16078 was merged, but I just saw a [revert PR](https://github.com/Expensify/App/pull/44096). Sooooo we may be waiting longer.

**Comment by rlinoz** [comment]
The revert doesn't fix the issue I am assigned to though, doest it fix yours?

**Comment by dangrous** [comment]
Oh nice! the revert was not actually merged, so #16078 is now on prod.

I just used the testing steps on staging and I think that this has been fixed... Can you confirm by testing on your ned @mallenexpensify ?

**Comment by mallenexpensify** [comment]
I was not able to reproduce, based on the steps in the OP. 

If I surface `@here` > click into a thread > click back buton, the pop up doesn't appear when I come back.  Thanks def not this bug, not sure if it's a bug at all (just sharing cuz I noticed) 

https://github.com/Expensify/App/assets/22508304/93b41a00-7dd0-40ec-8cc8-8ec3ee96eafe
